auth-service/src/main/kotlin/uk/gov/communities/delta/auth/controllers/internal/EditAccessGroupsController.kt

```diff
@@ -59,12 +123,67 @@ class EditAccessGroupsController(
 
         val accessGroupActions = generateAccessGroupActions(accessGroupRequestMap, currentAccessGroups, selectedOrgs)
 
-        executeAccessGroupActions(accessGroupActions, session, callingUser, call)
+        executeAccessGroupActions(accessGroupActions, null, callingUser, call)
 
         return call.respond(mapOf("message" to "Access groups have been updated. Any changes to your roles or access groups will take effect the next time you log in."))
     }
 
-    private suspend fun validateRequest(
+    fun validateAddAccessGroupRequest(
```

Comment:
I imagine the grant wizard also adds the user to `dclg` in the access group, can you check?

Reply:
We determined that it does but can you explain what you had in mind with this? After discussion with Hugh I removed the requirement that users not already be in/out of the group they are being added to/removed from.

Reply:
Nothing specific, but we'll need to make sure we keep that behaviour